src/core/load-fragment.c

```diff
@@ -2744,6 +2744,13 @@ int config_parse_environ(
                 return 0;
         }
 
+        const Specifier table[] = {
+                COMMON_SYSTEM_SPECIFIERS,
+                COMMON_TMP_SPECIFIERS,
+                COMMON_CREDS_SPECIFIERS(LOOKUP_SCOPE_USER),
+                {'h', specifier_user_home, NULL},
```

Comment:
Please insert a space after `{`, and before `}`.

What do you think about `specifier_user_shell`?

Reply:
Can't think of a use for it but I guess don't see why it shouldn't be available. I'll go ahead and add it too.